### PR TITLE
Auto-generate vite config for demos

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

**Review by toddburnside — APPROVED** [review]
Sweet! Looks good to me, but Carlos should probably check it. He's the vite expert.

**Comment by armanbilge** [comment]
Something funky going on, we're not getting a CI job for the PR, only the branch push.

**Comment by toddburnside** [comment]
> Something funky going on

Everything I do is gonna be funky, from now on - Dr. John

File: build.sbt [hunk 1]
```diff
@@ -20,6 +20,73 @@ lazy val yarnSettings = Seq(
...
+          |              _path.includes("/zinc/") ||
+          |              _path.includes("/classes");
+          |            return sjsIgnored;
+          |          },
+          |        ],
+          |      },
+          |    },
+          |    build: {
+          |      minify: 'terser',
+          |      // minify: false,
```
Comment (cquiroz) [inline build.sbt:75]: I guess the commented lines could be removed

**Review by cquiroz — APPROVED** [review]
This looks like a good idea, thanks

**Pull request merged**